rules-reviewed/jboss-eap/weblogic/weblogic.windup.xml

```diff
@@ -1,19 +1,186 @@
+                    <javaclass references="weblogic.transaction.TransactionManager.resume{*}">
+                        <location>METHOD_CALL</location>
+                    </javaclass>
+                    <javaclass references="weblogic.transaction.TransactionManager.forceResume{*}">
+                        <location>METHOD_CALL</location>
+                    </javaclass>
+                    <javaclass references="weblogic.transaction.ClientTransactionManager.resume{*}">
+                        <location>METHOD_CALL</location>
+                    </javaclass>
+                    <javaclass references="weblogic.transaction.ClientTransactionManager.forceResume{*}">
+                        <location>METHOD_CALL</location>
+                    </javaclass>
+                </or>
+            </when>
+            <perform>
+                <hint title="WebLogic TransactionManager Usage" effort="1" severity="mandatory">
+                    <message>Replace with the JEE standard javax.transaction.TransactionManager.resume(Transaction tx)</message>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="weblogic-06000">
+            <when>
+                <or>
+                    <javaclass references="weblogic.transaction.TransactionManager.suspend{*}">
+                        <location>METHOD_CALL</location>
+                    </javaclass>
+                    <javaclass references="weblogic.transaction.TransactionManager.forceSuspend{*}">
+                        <location>METHOD_CALL</location>
+                    </javaclass>
+                    <javaclass references="weblogic.transaction.ClientTransactionManager.suspend{*}">
+                        <location>METHOD_CALL</location>
+                    </javaclass>
+                    <javaclass references="weblogic.transaction.ClientTransactionManager.forceSuspend{*}">
+                        <location>METHOD_CALL</location>
+                    </javaclass>
+                </or>
+            </when>
+            <perform>
+                <hint title="WebLogic TransactionManager Usage" effort="1" severity="mandatory">
+                    <message>Replace with the JEE standard javax.transaction.TransactionManager.suspend()</message>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="weblogic-07000">
+            <when>
+                <javaclass references="weblogic.transaction.TxHelper">
+                    <location>IMPORT</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="WebLogic TxHelper Usage" effort="1" severity="mandatory">
+                    <message>Remove the 'weblogic.transaction.TxHelper' import statement.</message>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="weblogic-08000">
+            <when>
+                <javaclass references="weblogic.transaction.{classname}.getTransactionManager()">
```

Comment:
Yeah, I hope that this parameter stuff catches on. It's one of the more powerful new features in Windup 2.x, IMO.